.cargo/config.toml

```diff
@@ -1,9 +1,11 @@
 [target.'cfg(all(target_arch = "arm", target_os = "none"))']
 # Choose a default "cargo run" tool:
-# - probe-run provides flashing and defmt via a hardware debugger
+# - probe-run provides flashing and defmt via a hardware debugger, and stack unwind on panic
 # - elf2uf2-rs loads firmware over USB when the rp2040 is in boot mode
+# - "probe-rs-cli run" is similar to probe-run but it uses the latest probe-rs lib crate
```

Comment:
probe-rs-cli even has additional functionality, like GDB support. I have no idea how well that works and if it's worth mentioning here, though.